docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,290 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+You can use NGINX Ingress Controller fpr applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX Ingress Controleler in a istio service mesh environments without resorting to any workarounds. 
+
+
+Prior to 1.11 release, a configuration below would send two host headers to the backend; 
+
+```yaml
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: foo 
+spec:
+  host: foo.example.com
+  upstreams:
+  - name: foo
+    port: 8080
+    service: backend-svc
+  routes:
+  - path: "/"
+    action:
+      proxy:
+        upstream: foo
+        requestHeaders:
+          set:
+          - name: Host
+            value: bar.example.com
+```
+
+In 1.11 release, NGINX Ingress controller will only send one host header, depending on how you configure Ingress. By default NGINX Ingress Controller will send `proxy_set_header $host`. If Ingress has been configured with `requestHeaders` per the above example, this ensures that only one set of headers will be sent to the upstream server. In short, by setting `action-proxy-requestHeaders` in the `VirtualServer` CRD, NGINX Ingress will only send the specified hears that have been defined.    
+
+Here is a standard deployment of NGINX Ingress controller without any service mesh deployed:    
+
+{{< img src="./img/nginx_plain.png" alt="NGINX stand alone." >}}
+
+
+When deploying NGINX Ingress controller with Istio, you must deploy the envoy sidecar proxy into the NGINX Ingress controller pod.   
+The image below is what NGINX Ingress and Istio deployment looks like:    
+
+{{< img src="./img/nginx-envoy.png" alt="NGINX with envoy sidecar." >}}    
+
+
+By default for NGINX Ingress Controller, we populate the upstream server addresses with the endpoint IPs of the pods. NGINX Ingress Controller 1.11 release now supports the ability to configure NGINX Ingress CRDs (virtualServer/virtualServerRoute)to use the `service/cluster IP`. Using this flag,  NGINX Ingress will generate the .conf with the `service/cluster IP` of the service in the `upstreams/servers` section, instead of pod endpoint IPs of the pods.    
+
+To enable NGINX Ingress to route to the `Service IP`, we are going to use a new feature released in 1.11; [use-cluster-ip](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/#upstream).   

```

Comment:
Again.  the feature specific release comment is not relevant.  It should be mentioned as part of the above paragraph.